docs/components/query.md

```diff
@@ -460,6 +460,20 @@ Flags:
       --selector-label=<name>="<value>" ...
                                  Query selector labels that will be exposed in
                                  info endpoint (repeated).
+      --selector.relabel-config=<content>
+                                 Alternative to 'selector.relabel-config-file'
+                                 flag (mutually exclusive). Content of
+                                 YAML file that contains relabeling
+                                 configuration that allows selecting
+                                 blocks. It follows native Prometheus
+                                 relabel-config syntax. See format details:
+                                 https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config
+      --selector.relabel-config-file=<file-path>
+                                 Path to YAML file that contains relabeling
+                                 configuration that allows selecting
+                                 blocks. It follows native Prometheus
```

Comment:
Selecting blocks? :smile:

Reply:
I updated the description to make it more clear.